src/kademlia/node.cpp

```diff
@@ -623,6 +610,42 @@ time_duration node::connection_timeout()
 		}
 	}
 
+	int lifetime = m_settings.item_lifetime;
```

Comment:
please use time_duration here instead of int. that way, there won't be any question about its unit